src/QA/QA.Web/Views/Dashboard/Index.cshtml

```diff
@@ -74,7 +74,7 @@
                 <td>@vacancy.EmployerName</td>
                 <td>@vacancy.VacancyTitle</td>
                 <td><span>VAC</span>@vacancy.VacancyReference</td>
-                <td>@vacancy.SubmittedDate.AsGdsDateTime()</td>
+                <td><span asp-hide="vacancy.IsClosed">With employer</span><span asp-hide="vacancy.IsClosed">@vacancy.SubmittedDate.AsGdsDateTime()</span></td>
```

Comment:
Should both of these be asp-hide?

Reply:
Good catch... however this was not intended here... reverting back to original.